packages/ui/Scrollable/Scrollable.js

```diff
@@ -1051,7 +1051,7 @@ class ScrollableBase extends Component {
 		}
 
 		this.childRefCurrent.setScrollPosition(this.scrollLeft, this.scrollTop, this.props.rtl, ...rest);
-		this.forwardScrollEvent('onScroll');
+		this.forwardScrollEvent('onScroll', this.getReachedEdgeInfo());
```

Comment:
`forwardScrollEvent` might need to be updated to always include this object. I'm not clear on why only some events include it. Also, the docs for these events seem to be missing that this data is included as well.

cc: @ybsung